### DISCUSSION TITLE
1 New compliment not in the list

### DISCUSSION THREAD

**Discussion by Ant9cp** [Monika-After-Story/MonikaModDev#7784] [QA - tech support + general] [answered]
In the additions it says there's one new compliment but i don't see it in the i want to tell you something

**Comment by Booplicate** [comment, marked as answer]
There's a special condition for it because it doesn't make sense to always show it.

**Comment by Ant9cp** [comment]
What do you do to unlock it?

**Comment by Booplicate** [comment]
It's unlocked when you didn't open MAS for a while, then locked again until next time.

**Reply by WHATTHEDAN11** [reply]
the new compliment is 'i miss you', you can use it every few hours. Wouldn't make sense if u can repeat it over and over again... it would be annoying
